src/seismometer/controls/selection.py

```diff
@@ -39,57 +39,60 @@ def __init__(self, title: str, options: tuple[str], value: tuple[str] = None, sh
                 tooltip=str(option),
                 disabled=False,
                 button_style="",
-                layout=Layout(width="90%"),
+                layout=Layout(flex="1 0 auto"),
             )
             sub_toggle.observe(self._on_button_change, "value")
-            self.buttons.append(sub_toggle)
-            self.value_from_button[sub_toggle] = option
+            self.button_from_option[option] = sub_toggle
+            self.option_from_button[sub_toggle] = option
 
-        self.layout = Layout(width="max-content", min_width="100px")
-        self.label = Label(title)
-        if show_title:
-            self.children = [self.label] + self.buttons
+        self.layout = Layout(width="max-content", min_width="var(--jp-widgets-inline-label-width)")
+        self.button_box = VBox(
+            children=[button for option, button in self.button_from_option.items()],
```

Comment:
Since option isn't used, can we just do `children=[button for button in self.button_from_option.values()],` or make option an _ instead

Reply:
FINC - going with `children=[button for button in self.button_from_option.values()]`